<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrontIdentWidget</class>
 <widget class="QDialog" name="FrontIdentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>449</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>605</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Идентификация атмосферных фронтов</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>406</y>
     <width>168</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="addBtn">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="closeBtn">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>429</width>
     <height>388</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>Дата :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>Время:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="displayFormat">
           <string>hh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>Срок:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="hourCombo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Фактический</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>36</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>42</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>48</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>54</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>60</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>66</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>72</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>84</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>96</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>120</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>144</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>168</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>192</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>216</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>240</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Поля значений на уровне 850 мб</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="1">
       <widget class="QComboBox" name="fieldOT1500Combo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Нет данных</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="fieldH850Combo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Нет данных</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_21">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>OT 500/1000</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="fieldWindU850Combo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Нет данных</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_19">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Геопотенциальная высота</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_20">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Ветер U</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_18">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Температура</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="fieldT850Combo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Нет данных</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="fieldWindV850Combo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Нет данных</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_22">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Ветер V</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>По изотермам АТ-850</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Арктический фронт</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="SAcheckBox">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Старый арктический фронт</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="PFrSpin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-50.000000000000000</double>
           </property>
           <property name="maximum">
            <double>50.000000000000000</double>
           </property>
           <property name="value">
            <double>16.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="AFrSpin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-50.000000000000000</double>
           </property>
           <property name="maximum">
            <double>50.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="PcheckBox">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="AFcheckBox">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="SAFrSpin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-50.000000000000000</double>
           </property>
           <property name="maximum">
            <double>50.000000000000000</double>
           </property>
           <property name="value">
            <double>6.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Полярный фронт</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
